doc/Cifar10.rst

```diff
@@ -0,0 +1,1124 @@
+Estimating prediction sets on the Cifar10 dataset
+=================================================
+
+What is done in this tutorial ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Can you start with a short introductory sentence ? Something like "In this tutorial, we will build predictions sets ..."

Reply:
done